src/Layout/redist/targets/Directory.Build.targets

```diff
@@ -16,8 +16,8 @@
 
     <Import Project="BundledTemplates.targets" />
     <Import Project="Crossgen.targets" />
-    <Import Project="GenerateInstallerLayout.targets" />
-    <Import Project="GenerateArchives.targets" />
+    <Import Project="GenerateInstallerLayout.targets"  />
+    <Import Project="GenerateArchives.targets" Condition="'$(SkipBuildingInstallers)' != 'true'" />
```

Comment:
This seems like it got copied over by accident

Reply:
great spot!